asserter/construction.go

```diff
@@ -111,6 +111,7 @@ func ConstructionHashResponse(
 // not have a valid set of operations or
 // if the signers is empty.
 func (a *Asserter) ConstructionParseResponse(
+	signed bool,
```

Comment:
nit: can we do `response, signed` to keep consistency with all other Response validation functions?